app/src/main/java/com/kabouzeid/gramophone/util/PreferenceUtil.java

```diff
@@ -149,6 +151,16 @@ public final boolean coloredNotification() {
         return mPreferences.getBoolean(COLORED_NOTIFICATION, true);
     }
 
+    public final boolean classicNotification() {
+        return mPreferences.getBoolean(CLASSIC_NOTIFICATION, true);
```

Comment:
Shouldn't it be `return mPreferences.getBoolean(CLASSIC_NOTIFICATION, false);` as the preference is disabled by default (it is `defaultValue="false"` in pref_notification.xml)? Currently on fresh install it uses classic notification, while it is disabled in settings.